lib/ruby_indexer/lib/ruby_indexer/reference_finder.rb

```diff
@@ -35,10 +35,14 @@ class InstanceVariableTarget < Target
       #: String
       attr_reader :name
 
-      #: (String name) -> void
-      def initialize(name)
+      #: Array[String]
+      attr_reader :receiver_ancestors
+
+      #: (String name, Array[String] receiver_ancestors) -> void
+      def initialize(name, receiver_ancestors)
         super()
         @name = name
+        @receiver_ancestors = receiver_ancestors
```

Comment:
Let's call this `owner_ancestors` since instance variables don't technically have a receiver.